docs/tutorials/mobile/superres.md

```diff
@@ -0,0 +1,627 @@
+---
+title: Mobile super resolution
+description: Build Android and iOS applications for improving image resolution with built-in pre and post processing
+parent: Deploy on mobile
+grand_parent: Tutorials
+nav_order: 3
+---
+
+# Machine learning mobile application to improve image resolution
+{: .no_toc }
+
+Learn how to build an application to improve image resolution using ONNX Runtime Mobile, with a model that includes pre and post processing.
+
+You can use this tutorial to build the application for Android or iOS.
+
+## Contents
+{: .no_toc }
+
+* TOC placeholder
+{:toc}
+
+## Prepare the model
+
+The machine learning model used in this tutorial is based on the one used in the PyTorch tutorial referenced at the bottom of this page.
+
+We provide a convenient Python script that exports the PyTorch model into ONNX format and adds pre and post processing.
+
+1. Before running this script, install the following python packages:
+
+    ```bash
+    pip install pytorch
```

Comment:
>  pip install pytorch

is it `pip install pytorch` or `pip install torch`?

just tried at least on my machine, looks like the package installation name for pytorch is `torch`

Reply:
Fixed